miasm2/analysis/depgraph.py

```diff
@@ -28,7 +28,7 @@ class DependencyNode(object):
 
     def __init__(self, label, element, line_nb):
```

Comment:
In this whole object, `label` name is maybe not well suited

Reply:
done